### ISSUE TITLE
Remove pngquant

### ISSUE THREAD

**Issue by alippai** [gruntjs/grunt-contrib-imagemin#285]
It's duplicated (for every image optipng and pngquant is run and now there is no option for lossless compression) and it's removed from imagemin as well - it won't be supported in the next release.

**Comment by kevva** [comment]
Fixed and published.

**Comment by alippai** [comment]
Awesome!

**Comment by Boosten** [comment]
Question out of curiosity, why choose optipng instead pngquant? 

I was still at grunt-contrib-imagemin 0.9.2 when I updated my node packages it was automatically updated to 0.9.4. At that moment I noticed that my png images were not compressed as much as before. When I checked the changelog I saw that pngquant has been removed. 

When I take a look at a comparison between pngquant and optipng I see not that much of a difference (note: I'm not that experienced with compressing etc.)

pngquant average reduction / quality: 63.91% / 8.5
optipng average reduction / quality: 13.48% / 9.88
